client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -306,12 +306,25 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 		return ! isCopySiteFlow( flow );
 	};
 
-	const Container = [ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flow )
```

Comment:
Creating a React component on the flow forces React to rerender everything down the tree when the state changes. 

cc @leonardola

Reply:
@alshakero I'm not sure why I got pinged here. Did you mean to ping someone else maybe?

Reply:
He meantto ping @leonardost

Reply:
Yes hehe, thanks Osk! Thankfully, you're an Automattician as well.